AUTHORS

```diff
@@ -868,5 +868,125 @@ Super Zheng <superzheng@tencent.com>
+Richard Lau <riclau@uk.ibm.com>
+José Moreira <jose.moreira@findhit.com>
+Tom Atkinson <atkinson.tommy@nsoft.ba>
+Glen Arrowsmith <glen.arrowsmith@gmail.com>
+Jeff Harris <techjeffharris@gmail.com>
+Alexander Makarenko <estliberitas@gmail.com>
+Drew Folta <drew@folta.net>
+Myles Borins <myles.borins@gmail.com>
+Jimb Esser <wasteland@gmail.com>
+Dmitriy Lazarev <wKich@users.noreply.github.com>
+Adam Langley <agl@google.com>
+Kári Tristan Helgason <kthelgason@gmail.com>
+Manuel Valls <manolo@vlrz.es>
+Prayag Verma <prayag.verma@gmail.com>
+Gibson Fahnestock <gib@uk.ibm.com>
+Alan Cohen <alan@ripple.com>
+Christophe Naud-Dulude <christophe.naud.dulude@gmail.com>
+Matthias Bastian <dev@matthias-bastian.de>
+Phillip Kovalev <twilightfeel@gmail.com>
+Rainer Oviir <roviir@gmail.com>
+HUANG Wei <grubbyfans@gmail.com>
+DC <dcposch@dcpos.ch>
+Daniel Turing <mail@danielturing.com>
+Julie Pagano <julie.pagano@gmail.com>
+Ruben Bridgewater <ruben.bridgewater@fintura.de>
+Felix Becker <felix.b@outlook.com>
+Igor Klopov <igor@klopov.com>
+Tsarevich Dmitry <dimhotepus@users.noreply.github.com>
+Ojas Shirekar <ojas.shirekar@gmail.com>
+Noah Rose <noahroseledesma@seattleacademy.org>
+Rafael Cepeda <rcepeda1993@gmail.com>
+Chinedu Francis Nwafili <frankie.nwafili@gmail.com>
+Braydon Fuller <braydon@bitpay.com>
+Jiho Choi <jray319@gmail.com>
+Dirceu Pereira Tiegs <dirceutiegs@gmail.com>
+Marc Cuva <mjcuva@gmail.com>
+Chris Dew <cmsdew@gmail.com>
+Tristan Slominski <tristan.slominski@gmail.com>
+Hugo Wood <hx4.5@free.fr>
+Owen Smith <owen@omsmith.ca>
+Anton Andesen <a.andersen@readyforsky.com>
+Aayush Naik <aayushnaik17@gmail.com>
+Netto Farah <nettofarah@gmail.com>
+Craig Akimoto <strawbrary@users.noreply.github.com>
+Michael Barrett <mike182uk@gmail.com>
+Alexander Marchenko <axvmindaweb@gmail.com>
+Steve Mao <maochenyan@gmail.com>
+Robert C Jensen <robert.cole.jensen@gmail.com>
+Vaibhav <vaibhavbansal1993@gmail.com>
+Stefano Vozza <svozza@gmail.com>
+Daijiro Wachi <daijiro.wachi@gmail.com>
+Kirill Fomichev <fanatid@ya.ru>
+Mithun Patel <mithunster@gmail.com>
+Alexander Penev <alexander.penev@sap.com>
+Andrea Giammarchi <andrea.giammarchi@gmail.com>
+Lance Ball <lball@redhat.com>
+Jarrett Widman <jarrett@24m2.com>
+Florian Margaine <florian@margaine.com>
+Wolfgang Steiner <drywolf.dev@gmail.com>
+Bill Automata <billautomata@users.noreply.github.com>
+Robert Chiras <robert.chiras@intel.com>
+Corey Kosak <kosak@kosak.com>
+John Eversole <eversojk@gmail.com>
+Gareth Ellis <gareth.ellis@uk.ibm.com>
+Eugene Obrezkov <ghaiklor@gmail.com>
+Kyle Simpson <getify@gmail.com>
+Bogdan Lobor <b.lobor@outlook.com>
+Daniel Wang <wangyang0123@gmail.com>
+Mihai Potra <mike@mpotra.com>
+Brendon Pierson <BrendonPierson@gmail.com>
+Brad Hill <hillbrad@gmail.com>
+Mike Kaufman <mkaufman@microsoft.com>
+Igor Kalashnikov <igor.kalashnikov@me.com>
+Anna Henningsen <anna@addaleax.net>
+Amery <amery@xiangfa.org>
+James Reggio <james.reggio@gmail.com>
+Imran Iqbal <imran@imraniqbal.org>
```

Comment:
0bf9a7c758c80d370b9c57f65800196c083bfc38 and 8b57b312996794526a8d5efe5935b64ba412c4e6. Yes, duplicates.
/cc @iWuzHere